jbehave-core/src/main/java/org/jbehave/core/reporters/JsonOutput.java

```diff
@@ -177,6 +176,7 @@ protected String format(String key, String defaultPattern, Object... args) {
         if ("afterExamples".equals(key) && !stepPublishing) {
             print("}");
         }
+        ScenarioStatus scenarioStatus = getCurrentScenarioStatus();
```

Comment:
not the best way to cache, the conditions might be changed and cached value will become invalid